src/coreclr/src/inc/pinvokeoverride.h

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*****************************************************************************
+ **                                                                         **
+ ** pinvokeoverride.h - PInvoke binding override                            **
+ **                                                                         **
+ *****************************************************************************/
+
+#ifndef _PINVOKEOVERRIDE_H_
+#define _PINVOKEOVERRIDE_H_
+
+#ifndef _MSC_VER
+#define __stdcall
+#endif
+
+typedef const void* (__stdcall PInvokeOverrideFn)(const char* libraryName, const char* entrypointName);
```

Comment:
This should use platform-default calling convention. We do not want to use `__stdcall` on Unix x86.

Reply:
I assumed x86 needs this since `BundleProbe` is __stdcall .
Should we drop it there too?

Reply:
The stdcall is set as default globally (look for `/Gz` under `eng/...`). We prefer to not specify it explicitly unless it is really required.